### ISSUE TITLE
American Providers BOM

### ISSUE THREAD

**Issue by durinwinter** [esa-prl/ExoMy#9]
We have sourced everything from Sparkfun and some of the parts change, am curious if we can upstream wiki/BOM changes once we have everything resolved.

**Comment by mvoellmy** [comment]
Hey @durinwinter! That sounds like a good idea.
I suggest to add a second column to `PI_ExoMy_Purchasing_BoM.xlsx` containing the US suppliers.
Once it's finalized, please attach it here and I can update it in the release and make sure the link in the wiki points to the new one.

**Comment by maxehrhardt** [comment]
@durinwinter Is there any update on the BOM?

**Comment by durinwinter** [comment]
yes! I have it - one thing I am missing is the dcdc 11.5v to 5v. but
besides that I have it.
<https://www.winterwinds.io/>
Caleb Eastman

Co-Founder, C3PO
M:  970.632.1890
P:  303.551.0570
E:  caleb@winterwinds.io
A: 1910 Pike Rd Unit A, Longmont, CO 80501
www.winterwinds.io
<https://www.linkedin.com/in/caleb-eastman-55465352/>
<https://twitter.com/winterwindsr>
<https://www.instagram.com/winterwindsrobotics/>
<https://www.facebook.com/winterwindsrobotics/>
<https://mysignature.io/editor/?utm_source=expiredpixel>

**Comment by durinwinter** [comment]
[ExoMy_Purchasing_BoM.xlsx](https://github.com/esa-prl/ExoMy/files/6064703/ExoMy_Purchasing_BoM.xlsx)
